ci-operator/config/hybrid-cloud-patterns/docs/hybrid-cloud-patterns-docs-main.yaml

```diff
@@ -0,0 +1,30 @@
+build_root:
+  image_stream_tag:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+promotion:
```

Comment:
using promotion is meant to push an image into OpenShift CI's internal registry for post-submit jobs. I could be mistaking but I don't believe this is needed here.